### PR TITLE
Update resource in deployment manifests with G/V/R format

### LINKED ISSUE #331: [BUG] BotKube resource configuration in deployment manifests pointing to older syntax
**Describe the bug**

Update default resource configuration in helm chart and deploy-all-in-one yamls wrt #253  
https://github.com/infracloudio/botkube/blob/develop/helm/botkube/values.yaml#L33
https://github.com/infracloudio/botkube/blob/develop/deploy-all-in-one.yaml#L16

### PR DESCRIPTION
**Pull request by PrasadG193**
Signed-off-by: Prasad Ghangal <prasad.ghangal@gmail.com>

##### ISSUE TYPE

 - Bug fix Pull Request

##### SUMMARY

Update resource in deployment manifests with G/V/R format

Fixes #331

### CODE REVIEW

**Comment by rahulchheda** [comment]
In G/V/R only supports prural resource(eg: pod -> v1/pods)

**Comment by rahulchheda** [comment]
I see a lot of indentation changes, are we using some linting tool for this?

File: deploy-all-in-one.yaml [hunk 1]
```diff
@@ -16,190 +16,190 @@ data:
...
+      namespaces:
+        include:
+          - all
+        ignore:
+          -
+      events:
+        - create
+        - delete
+        - error
+    - name: rbac.authorization.k8s.io/v1/clusterrolebindings
```
Comment (rahulchheda) [inline deploy-all-in-one.yaml:183]:
```suggestion
    - name: rbac.authorization.k8s.io/v1/clusterroles
```

File: deploy-all-in-one-tls.yaml [hunk 2]
```diff
@@ -16,190 +16,190 @@ data:
...
+      namespaces:
+        include:
+          - all
+        ignore:
+          -
+      events:
+        - create
+        - delete
+        - error
+    - name: rbac.authorization.k8s.io/v1/clusterrolebindings
```
Comment (rahulchheda) [inline deploy-all-in-one-tls.yaml:183]:
```suggestion
    - name: rbac.authorization.k8s.io/v1/clusterroles
```

**Comment by PrasadG193** [comment]
@rahulchheda Yes, we run helm lint. Basically for lists `  ` spaces are optional. Removing it won't affect anything but we are already using an additional 2 spaces in templates. Let's keep the consistency

**Review by sanketsudake — APPROVED** [review]

**Pull request merged**